core/templates/dev/head/pages/exploration-editor-page/preview-tab/preview-tab.directive.ts

```diff
@@ -77,7 +77,7 @@ angular.module('oppia').directive('previewTab', [
             UrlInterpolationService) {
           var ctrl = this;
           ctrl.isExplorationPopulated = false;
-          ExplorationDataService.getData().then(function() {
+          ExplorationDataService.getData($.noop, true).then(function() {
```

Comment:
Does ExplorationDataService store a copy of the updated exploration (while changes are made in an editing session)? If so, I would expect to be using that instead -- we should not be triggering a separate fetch and relying on the backend draft functionality. Instead, it might be worth investigating why that copy of the exploration does not get updated.